packages/wdio-browserstack-service/src/util.ts

```diff
@@ -638,14 +638,140 @@ export function getCiInfo () {
         }
     }
     // Visual Studio Team Services
-    if (env.TF_BUILD === 'True') {
+    if (isTrue(env.TF_BUILD)) {
         return {
             name: 'Visual Studio Team Services',
             build_url: `${env.SYSTEM_TEAMFOUNDATIONSERVERURI}${env.SYSTEM_TEAMPROJECTID}`,
             job_name: env.SYSTEM_DEFINITIONID,
             build_number: env.BUILD_BUILDID
         }
     }
+    // Appveyor
+    if (isTrue(env.APPVEYOR)) {
+        return {
+            name: 'Appveyor',
+            build_url: `${env.APPVEYOR_URL}/project/${env.APPVEYOR_ACCOUNT_NAME}/${env.APPVEYOR_PROJECT_SLUG}/builds/${env.APPVEYOR_BUILD_ID}`,
+            job_name: env.APPVEYOR_JOB_NAME,
+            build_number: env.APPVEYOR_BUILD_NUMBER
+        }
+    }
+    // Azure CI
+    if (env.AZURE_HTTP_USER_AGENT && env.TF_BUILD) {
```

Comment:
Already defined at line 641

Reply:
That one is for Team Foundation, this is for Azure CI. [Ref](https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/ci_provider.js#L23-L34) from cypress

Reply:
I see, in that case we will have to add TF_BUILD_BUILDNUMBER or there will by no distinction between the two ;)

Reply:
Added